app/models/comment.rb

```diff
@@ -1,2 +1,4 @@
 class Comment < ApplicationRecord
```

Comment:
- [ ] `Comment` model should have method that updates comments counter for a post. This should be similar to the example below:

```ruby
# app/models/comment.rb
...

def update_comments_counter
  post.increment!(:comments_counter)
end
```

Reply:
Done